docs/src/data/hid.js

```diff
@@ -7865,4 +7865,25 @@ export default [
     },
     footnotes: {},
   },
+  {
+    names: ["C_KEYBOARD_LAYOUT_SELECT", "GLOBE"],
+    description: "Layout Select (Apple Globe)",
+    context: "Consumer AC",
+    clarify: true,
+    usages: [
+      {
+        application: consumerApplication,
+        item: usage(consumerPage, 0x29d),
+      },
+    ],
+    documentation: "https://usb.org/sites/default/files/hut1_2.pdf#page=150",
+    os: {
+      windows: null,
+      linux: true,
+      android: null,
+      macos: true,
+      ios: null,
```

Comment:
```suggestion
      windows: null,
      linux: null,
      android: null,
      macos: true,
      ios: true,
```
Just guessing this might be the case here, in case you copy pasted it and didn't modify this section?

Reply:
Yep you're right, my bad